lively.morphic/morph.js

```diff
@@ -1642,6 +1641,11 @@ export class Morph {
     return this;
   }
 
+  delete () {
```

Comment:
Just to make sure I got Silvans explanation correctly:
The idea behind this method is that with your changes, every place that deletes Morphs that we know of calls delete. In the case that in the future someone wants to do something in a subclass of morph before an object of the class is deleted, one can overwrite the delete method and and call super.delete in the end, providing a best effort solution to the "do-something-before-deletion" problem?

Reply:
that is correct since there is no seperation at the moment between deleting and moving so it is hard to do something before deletion but not always if a morph ist grabbed